--- a/hardware/nvidia/soc/t210/kernel-dts/tegra210-soc/tegra210-soc-base.dtsi	2020-12-12 12:52:58.035090277 +0100
+++ b/hardware/nvidia/soc/t210/kernel-dts/tegra210-soc/tegra210-soc-base.dtsi	2020-12-12 12:55:29.686078340 +0100
@@ -351,7 +351,10 @@
 		#interrupt-cells = <3>;
 		interrupt-controller;
 		reg = <0x0 0x50041000 0x0 0x1000
-		       0x0 0x50042000 0x0 0x0100>;
+		       0x0 0x50042000 0x0 0x0100
+		       0x0 0x50044000 0x0 0x2000
+		       0x0 0x50046000 0x0 0x2000>;
+		interrupts = <0x1 0x9 0xff04>;
 		status = "disabled";
 	};
 
